### PR TITLE
Update to document around agent/server run by a local user account on Windows

### PR DESCRIPTION
**Pull request by anishvenkat**

### CODE REVIEW

File: installation/install/server/windows.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ C:\> go-server-16.1.0-1234-setup.exe /S /D=C:\go\server
     wrapper.java.additional.17=-Dcruise.config.other=baz
     ```
 
+    **Please note** : If the go-agent as an application is run by a local user account, then this user needs to have these required permissions to the go-agent folder, i.e. modify, read and execute, list folder contents and read permissions.
```
Comment (ketan) [inline installation/install/server/windows.md]: did you mean to say "go-server"?

File: installation/install/agent/windows.md [hunk 2]
```diff
@@ -61,6 +61,7 @@ C:\> go-agent-16.1.0-1234-setup.exe /S /SERVERURL="https://10.12.20.47:8154/go"
     wrapper.java.additional.16=-Dcruise.config.foo=bar
     wrapper.java.additional.17=-Dcruise.config.other=baz
     ```
+    **Please note** : If the go-agent as an application is run by a local user account, then this user needs to have these required permissions to the go-agent folder, i.e. modify, read and execute, list folder contents and read permissions.
```
Comment (varshavaradarajan) [inline installation/install/agent/windows.md]: Its local system account.

Comment (varshavaradarajan) [inline installation/install/agent/windows.md]: Sorry, this seemed a little confusing to me. You meant to say that the regular user must have the right permissions, that is the permissions that the local system account has to access the go-agent directory, right?

File: installation/install/server/windows.md (hunk 1 above)
Comment (anishvenkat) [inline installation/install/server/windows.md]: Yes Ketan, I'll quickly add that.

File: installation/install/agent/windows.md (hunk 2 above)
Comment (anishvenkat) [inline installation/install/agent/windows.md]: Yeah, it's the local system account. I'll update the change.

Comment (varshavaradarajan) [inline installation/install/agent/windows.md]: So, saying `If the go-agent as an application is run by any user...` should suffice.

**Pull request closed without merging**